2-projects/1-todo-list/script.js

```diff
@@ -1,24 +1,63 @@
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
+   let list = document.getElementById("todo-list");
+ 
+  
+      let liEl=document.createElement('li')
+      liEl.className='list-group-item d-flex justify-content-between align-items-center'
+      liEl.innerHTML=todos[todos.length-1].task
+
+      let spanEl=document.createElement('span')
+      spanEl.className=('badge bg-primary rounded-pill')
+
+      liEl.appendChild(spanEl)
+      list.appendChild(liEl)
+
+      let checkiEl=document.createElement('i');
+      checkiEl.className= 'fa fa-check';
+      checkiEl.ariaHidden;
```

Comment:
Lines 17 and 28 accesses the property ariaHidden but don't seem to have a use. Maybe the intention was to set the property to values?